docs/user/bots.rst

```diff
@@ -945,12 +945,15 @@ Install the `stomp.py` library from PyPI:
 **Configuration Parameters**
 
 * **Feed parameters** (see above)
-* `exchange`: exchange point
+* `exchange`: exchange point, e.g. "/exchange/my.org/*.*.*.*"
 * `port`: 61614
-* `server`: hostname e.g. "n6stream.cert.pl"
+* `server`: hostname e.g., "n6stream.cert.pl"
 * `ssl_ca_certificate`: path to CA file
-* `ssl_client_certificate`: path to client cert file
-* `ssl_client_certificate_key`: path to client cert key file
+auth_by_ssl_client_certificate`: Boolean, default: true (note: set to false for new *n6* auth)
```

Comment:
```suggestion
* `auth_by_ssl_client_certificate`: Boolean, default: true (note: set to false for new *n6* auth)
```

Reply:
Fixed.

Reply:
Note: now I added also `CHANGELOG.md`.

Reply:
Now: a few corrections in `CHANGELOG.md`.

Reply:
Now (among others): more corrections/additions in `CHANGELOG.md`.